### DISCUSSION TITLE
Blog titles not showing on github pages home page when using a remote theme...

### DISCUSSION THREAD

**Discussion by iankingcoop** [skills/.github#332] [GitHub Pages]
I am trying to use a remote theme (midnight) but my page breaks (no blog titles shown) when I use that theme. I see no specific steps to take in the repos for the various themes, other than updating my _config.yml file.

So Im not sure what to do..

**Comment by sinsukehlab** [comment]
How would you like your blog titles?
https://iankingcoop.github.io/cyberian_blog/

**Comment by 0ldcamel** [comment]
I have the same question, hoping for an answer soon.
